webview/src/setup/components/experiments/Experiments.tsx

```diff
@@ -43,12 +44,17 @@ export const Experiments: React.FC<ExperimentsProps> = ({ isDvcSetup }) => {
   return (
     <EmptyState isFullScreen={false}>
       <h1>Setup Complete</h1>
+      <p>
+        Learn more about using <a href="https://dvc.org/doc/dvclive">DVCLive</a>{' '}
+        in your project in its docs.
```

Comment:
[C] Maybe change this to have the link on `the docs`.

Reply:
or `Learn more about using DVCLive in the docs` with the link on `docs`.

Reply:
Updated!